packages/node-integration-tests/utils/index.ts

```diff
@@ -102,12 +102,16 @@ export async function runScenario(url: string): Promise<void> {
   await Sentry.flush();
 }
 
-async function makeRequest(method: 'get' | 'post' = 'get', url: string): Promise<void> {
+async function makeRequest(
+  method: 'get' | 'post' = 'get',
+  url: string,
+  axiosConfig?: AxiosRequestConfig,
+): Promise<void> {
   try {
     if (method === 'get') {
-      await axios.get(url);
+      await axios.get(url, axiosConfig);
     } else {
-      await axios.post(url);
+      await axios.post(url, axiosConfig);
```

Comment:
thoughts on this approach for adding axois config @onurtemizkan? Should we do it some other way? Maybe by putting base axios config in the `TestEnv` constructor?

Reply:
Test 123

Reply:
I think this approach (passing to the data method) with a default header set in `TestEnv` would make it easier if we want to test consecutive requests with different configurations?

I have just tested setting the default inside class, and it's not breaking any of the current Node / Remix tests.

Reply:
I extract this to be a setter on the class, lmk what you think.

Reply:
Looks good to me!